### DISCUSSION TITLE
support rtl?

### DISCUSSION THREAD

**Discussion by sinafath** [tamagui/tamagui#674] [General]
does  tamagui support rtl? do you have any plan to support it?

**Comment by redbar0n** [comment]
I presume you mean RTL as in right-to-left layout.

I believe Tamagui supports it, as a part of the localization in React Native for Web, since Tamagui uses that (or potentially an even lighter version of it, [react-native-web-lite](https://github.com/tamagui/tamagui/tree/master/packages/react-native-web-lite), if you enable it).

Localization support like RTL is not documented the best at the Tamagui docs itself, but see: https://tamagui.dev/docs/intro/props which says:

> You may also want to reference the [react-native-web docs](https://necolas.github.io/react-native-web/docs/) for more on props related to:
>  - ...
>  - [Localization ](https://necolas.github.io/react-native-web/docs/localization/)
>
>Tamagui adds a few extra props, ...

RTL is documented on the linked React Native for Web docs: https://necolas.github.io/react-native-web/docs/localization/

See also this [CodePen demo for RTL in React Native Web](https://codepen.io/necolas/pen/BzbRzz).

**Reply by sinafath** [reply]
yeah thanks  but I use  next.js and padding start and end [doesn't work](https://github.com/tamagui/tamagui/issues/675#issue-1574291608) in[ next-expo-solito](https://github.com/tamagui/tamagui/tree/master/starters/next-expo-solito) .
